src/components/dashboard/PendingTxs/styles.module.css

```diff
@@ -0,0 +1,48 @@
+.gridContainer {
+  width: 100%;
+  display:grid;
+  grid-gap: 4px;
+  align-items: center;
+  padding: 12px 16px;
```

Comment:
This was `8px 16px` before which made the accordion look slightly smaller. Is this change intended?

Reply:
Great eye ser! Also not intended